dali/pipeline/data/tensor_vector.h

```diff
@@ -113,6 +113,22 @@ class TensorVector {
     return tensors_.size();
   }
 
+  size_t nbytes() const noexcept {
+    size_t total_nbytes = 0;
+    for (const auto &t : tensors_) {
+      total_nbytes += t->nbytes();
+    }
+    return total_nbytes;
+  }
+
+  size_t capacity() const noexcept {
+    size_t total_capacity = 0;
+    for (const auto &t : tensors_) {
+      total_capacity += t->capacity();
+    }
+    return total_capacity;
```

Comment:
Those functions need to be handle similarly to, for example shape(), so
```
if (state_ == State::contiguous) {
  return tl->nbytes();
} 
size_t total_nbytes = 0;
for (const auto &t : tensors_) {
  total_nbytes += t->nbytes();
}
return total_nbytes;
```
as you can have them backed by tensor list or vector of tensors.

Reply:
Done